CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0]
+
+### Summary
+The addition of several OS support features and a considerable amount of compatibility and bug fixes. 
+
+### Known issues
+While this release adds considerable features and bug fixes the following areas are known issues and require more work:
+- For Windows, trigger an agent run to get Puppet to create the nessesary directory structures.
```

Comment:
Here and below I would add 'after upgrade': 'trigger an agent run after upgrade'

Could also caveat that to `puppet resource` calls will fail until the first puppet agent run.  (You really don't need a manual puppet agent run unless you have to call puppet resource prior to the first daemonized agent run...if it's worth splitting the hair).